dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/build.gradle

```diff
@@ -10,7 +10,24 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
+apply from: '../wlp-gradle/subprojects/maven-central-mirror.gradle'
+	
+configurations {
+  shimLibs
+}
 
 dependencies {
   requiredLibs project(':io.openliberty.mpTelemetry.1.0.thirdparty')
-} 
+  shimLibs 'io.opentelemetry:opentelemetry-api:1.19.0',
```

Comment:
I don't think you want the opentelemetry-api in `shimLibs`

Although the shim does depend on the otel API, as far as I can see you're using `shimLibs` to add the libraries to the test application. The test application shouldn't need the otel API since that's provided for it by liberty.